app/models/services/no-op-service.ts

```diff
@@ -63,6 +63,7 @@ export default class NoOpService extends BaseService<void> {
       links,
       request: getRequestUrl(req),
       numInputGranules: this.operation.cmrHits,
+      destination_url: this.operation.destinationUrl,
```

Comment:
This will return a destination url for the no op service. I think we should eventually get rid of the no op service, but in any case since we aren't doing anything and won't be staging anything to a bucket we should remove this line.